department-of-veterans-affairs/caseflow
# Add/Edit and Delete Hearing Day Feature Tests

Resolves #8595 

### Description
Added feature tests to cover the Add, Edit and Remove Hearing Days functionality.

### Acceptance Criteria

### Testing Plan
1. Execute the feature tests
2. Verify they all go green
